acknowledge_license_cert.html

```diff
@@ -11,28 +11,35 @@
 
 <h3>License information</h3>  
 <div class="paragraph">   
-Unless otherwise stated, this code is subject to the terms of the <a href="license.txt">
-GNU General Public License version 3 or higher (GPL)</a>, a copy of which is included here
- for your convenience.
+To be announced
```

Comment:
I think you have to say that the code is GPL, since it is and we can't change that with out the authors permission.  In the future we can add a line about the code that is Apache or whatever it ends up being